script/UpgradePuffETH.s.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
@bxmmm1 is there a test that shows this upgrade works as expected? E.g. we can call the upgrade on the vault and try calling some of the functions in a test

Reply:
https://github.com/PufferFinance/pufETH/blob/feature/mainnet/test/Integration/PufferTest.integration.t.sol#L362